source/docs/software/wpilib-tools/robotbuilder/advanced/robotbuilder-command-run-commands.rst

```diff
@@ -0,0 +1,72 @@
+Creating a Command that Runs Other Commands
+===========================================
+
+Often you will want to run multiple commands, one after another to enable more complex behaviors in your program. Once each of the individual commands have been debugged, you can create a CommandGroup. A CommandGroup is a named set of commands that may be executed sequentially or in parallel.
+
+Creating a Command Group
+------------------------
+
+.. image:: images/command-run-commands-1.png
+
+To create a CommandGroup:
+
+1. Drag the command group from the palette to Commands folder in the robot description
+2. Name the command group so that is has a meaningful name
+3. Give the group any parameters it needs and presets for those parameters, just like with a normal command. These can be changed later at any time
+
+Overview of the Editor
+----------------------
+
+.. image:: images/command-run-commands-2.png
+
+The elements of the command group editor are:
+
+1. The properties of the command group.
+2. The editor where commands are placed and connected.
+3. The organize button will automatically organize all the commands in the editor
+
+There are also a few differences between sequential and parallel commands in the editor.
+
+- A command running sequentially can have only one command run sequentially after it
+- A command running in parallel cannot have any commands run after it
+- Sequential commands are colored light green and are placed at the top of the editor when it's organized
+- Parallel commands are colored light blue and prefer to be placed below sequential commands
+
+Adding Commands
+---------------
+
+.. image:: images/command-run-commands-3.png
+
+An empty command group isn't very useful, so to add commands to it:
+
+1. Click a command in the palette ("Command 1" in this example)
+2. Drag it somewhere into the graph editor
+
+Alternatively, you can right-click anywhere on the editor and choose the command from the dropdown, or right click a command and choose a command to automatically be placed after the command you click.
+
+Now you have a command in the group! But it's not connected to anything, so RobotBuilder doesn't know how it's supposed to behave.
+
+Connecting Commands
+-------------------
+
+.. image:: images/command-run-commands-4.png
+
+To connect commands and control when commands run in the group,
+
+1. Click the center of the box for the previous command (or "Start" if you want it to run at the beginning). In this example, "Command 1" is the first command to run "Start" is clicked.
+2. When a yellow box appears around the first box, drag your mouse to the command to run after it. This will create an arrow from the first box to your mouse
```

Comment:
```suggestion
2. When a yellow box appears around the first box, drag your mouse to the command to run after it. This will create an arrow from the first box to your mouse.
```